composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=5.4",
-    "moxie-lean/wp-endpoint": "1.*.*",
+    "moxie-lean/wp-endpoint": "2.*.*",
```

Comment:
Sounds good.

I think we should update other packages as well to use 

`^2.0` 

In that way we can control all of this things.

> On May 31, 2016, at 1:27 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In composer.json <https://github.com/moxie-lean/wp-endpoints-static/pull/26#discussion_r65237197>:
> 
> > @@ -17,7 +17,7 @@
> >    ],
> >    "require": {
> >      "php": ">=5.4",
> > -    "moxie-lean/wp-endpoint": "1.*.*",
> > +    "moxie-lean/wp-endpoint": "2.*.*",
> I agree, this is the standard way of doing it. The problem is we already have several packages using the * way... so using ^ here will make this package inconsistent and (if i remember rightly) causes issues between them.
> 
> We can open a task to go through all active packages to fix this.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/moxie-lean/wp-endpoints-static/pull/26/files/c8d9a781d5a0df383fcbd228d9bd4773fe876dce#r65237197>, or mute the thread <https://github.com/notifications/unsubscribe/ADvViYrAmwMmEKuSQfsFYg5_0duuU1thks5qHH2RgaJpZM4IqzTs>.
>